packages/light-client/src/client/index.ts

```diff
@@ -70,16 +69,7 @@ export class Lightclient {
     const header = headerResp.data.header.message;
     const stateRoot = header.stateRoot;
 
-    // fetch a proof with everything needed to bootstrap the client
-    const paths = [
-      // initial sync committee list
-      ...getSyncCommitteesProofPaths(),
-      // required to initialize a slot clock
-      ["genesisTime"],
-      // required to verify signatures
-      ["genesisValidatorsRoot"],
-    ];
-    const proof = await api.lightclient.getStateProof(toHexString(stateRoot), paths);
```

Comment:
Should we assert that state.genesisTime and state.genesisValidatorsRoot are not zero explicitly as a sanity check?

Reply:
I don't think they'll ever be zero. If the proof doesn't contain the data, an error will be thrown rather than zeros being returned.